.pylintrc

```diff
@@ -76,6 +76,7 @@ disable=attribute-defined-outside-init,
         no-member,
         protected-access,
         R,
+        unidiomatic-typecheck,
```

Comment:
Why? Wouldn't `isinstance(foo, str)` work?

Reply:
I don’t know why, but it didn’t. The type was Unicode in Circle and it was best caught by checking for string. If I added Unicode to the isinstance list it would skip over and return None for the state. But this works.